aiobotocore/client.py

```diff
@@ -63,7 +97,8 @@ async def _create_client_class(self, service_name, service_model):
         await self._event_emitter.emit(
             'creating-client-class.%s' % service_id,
             class_attributes=class_attributes,
-            base_classes=bases)
+            base_classes=bases,
```

Comment:
I've noticed this in a few places an extra comma, mind cleaning them up?  or is this copied from botocore?

Reply:
saw it came from botocore, sigh